tests/Core/Header/FluentHeaderTests.razor

```diff
@@ -0,0 +1,57 @@
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    [Inject]
+    private LibraryConfiguration LibraryConfiguration { get; set; } = new LibraryConfiguration();
+
+    public FluentHeaderTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddSingleton(LibraryConfiguration);
+    }
+    [Fact]
+    public void FluentHeader_Default()
+    {
+        //Arrange
+        var childContent = "<b>render me</b>";
```

Comment:
Please, don't use variable in Unit Tests, but set the value directly in ´Add´ and ´AddChildren´ methods.

Reply:
Ok so these other 28 all need updated as well.  I might do that this weekend. In another PR.  Can we just approve this one as is and I will fix them all at once?
![image](https://github.com/microsoft/fluentui-blazor/assets/357219/85ec4520-12ae-46e5-9332-5e42aafe8b08)

Reply:
Searching I could find only one test that sets it directly. If you want to change this, I can fix them all at once. But not until the weekend.

Reply:
But I did update this one.

Reply:
I'll approve this one, but please update the others as well

Reply:
I'm not necessarily asking you to modify our old code, we can do that again during a future refactoring session. But you can take care of it :-)
My comment was more about your code, which could use better practices than what we did at the start :-)